src/starbunk/bots/botRegistry.ts

```
@@ -13,12 +17,186 @@ export class BotRegistry {
+					// This approach avoids using file:// URLs which can cause issues with require() in the compiled JS
+					const botModule = await import(absolutePath);
+					const bot = botModule.default;
+
+					// Validate the bot
+					if (!bot || typeof bot !== 'object') {
+						logger.warn(`[BotRegistry] Invalid bot module in directory: ${botDir}`);
+						continue;
+					}
+
+					// Check if it's a valid ReplyBotImpl
+					if (!this.validateBot(bot)) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)